### PR TITLE
Annual npm vulnerabilities audit

### PR DESCRIPTION
**Pull request by shulkaolka**

### CODE REVIEW

File: lib/actions/insert.js [hunk 1]
```diff
@@ -26,16 +26,16 @@ function init(cfg) {
...
     // Always attach an error listener
-    connection.on('error', (err) => this.emit('error', err));
+    connection.on('error', err => this.emit('error', err));
     let sql = cfg.query;
     yield connection.connect();
     logger.info('Connection established');
-    logger.trace('Preparing query=%s', sql);
+    logger.debug('Preparing query...');
     const vars = sql.match(VARS_REGEXP);
     logger.trace('Found following prepared variable:type pairs=%j', vars);
```
Comment (denyshld) [inline lib/actions/insert.js]: Do this pairs contains some vulnerable data?

**Review by A3a3e1 — APPROVED** [review]

File: lib/actions/insert.js (hunk 1 above)
Comment (shulkaolka) [reply lib/actions/insert.js]: I tried to run action, in logs:
`Found following prepared variable:type pairs=["@lang","@retweeted:boolean","@favorited:boolean","@text:string","@id:bigint","@created_at:date","@username","@screenname:string"]`
@denyshld @A3a3e1 thoughts? Are column names and types sensitive data?

Comment (denyshld) [reply lib/actions/insert.js]: For all table - yes

Comment (A3a3e1) [reply lib/actions/insert.js]: I don't think it is sensitive data

**Review by denyshld — APPROVED** [review]

**Pull request merged**